apiserver/params/crossmodel.go

```diff
@@ -264,9 +264,10 @@ type RemoteRelationResults struct {
 // RemoteApplication describes the current state of an application involved in a cross-
 // model relation, from the perspective of the local environment.
 type RemoteApplication struct {
-	Name   string `json:"name"`
-	Life   Life   `json:"life"`
-	Status string `json:"status"`
+	Name      string `json:"name"`
+	Life      Life   `json:"life"`
+	Status    string `json:"status"`
+	ModelUUID string `json:"model-uuid"`
```

Comment:
Yeah, in this file the structs tended to have attribute comments, which makes it a PITA to add new structs. Elsewhere we don't bother. I'll add comments for consistency.